spec/models/post_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  describe 'validations' do
+    it { should validate_presence_of(:title) }
+    it { should validate_length_of(:title).is_at_most(250) }
+    it { should validate_numericality_of(:comments_counter).only_integer.is_greater_than_or_equal_to(0) }
+    it { should validate_numericality_of(:likes_counter).only_integer.is_greater_than_or_equal_to(0) }
+  end
+
+  describe 'associations' do
+    it { should belong_to(:author).class_name('User') }
+    it { should have_many(:comments).dependent(:destroy) }
+    it { should have_many(:likes).dependent(:destroy) }
+  end
+end
```

Comment:
- [ ] Please fix tests that are failing
<img width="1165" alt="Screen Shot 2023-06-24 at 21 33 13" src="https://github.com/matee-safi/Blog-app/assets/44745855/2e81fea3-bee8-4549-b803-f1072fb5d61e">

Reply:
- [ ] Well resolved 👍🏽 
<img width="406" alt="Screenshot 2023-06-24 at 21 05 33" src="https://github.com/matee-safi/Blog-app/assets/58692590/38c7a2ad-328e-4064-8062-e501e4944a0c">